frontend/uddhar-frontend/src/organization/components/VolunteerReqList.jsx

```
@@ -108,100 +106,97 @@ const VolunteerReqList = () => {
+                  <td className="p-3">{user.location}</td>
+                  <td className="p-3 max-w-30 overflow-hidden">{skillsText}</td>
+                  <td className="p-3 flex justify-center items-center">
+                    <button
+                      disabled={user.status === "approved"}
+                      onClick={() => handleStatusChange(user.id, "approved")}
+                      className={`border-2 border-green-800 rounded-[5px] bg-green-200 hover:bg-green-700 hover:text-white p-1 text-gray-800 mr-1 ${
+                        user.status === "approved"
+                          ? "opacity-50 bg-green-700 text-white font-semibold cursor-not-allowed"
+                          : "cursor-pointer"
+                      }`}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)